Gems/Atom/Utils/Code/Source/ImGuiGpuProfiler.cpp

```diff
@@ -1535,6 +1684,242 @@ namespace AZ
             }
         }
 
+        static constexpr const char* MemoryCSVHeader =
+            "\"Pool Name\", \"Memory Type (0 == Host : 1 == Device)\", \"Allocation Name\", \"Allocation Type (0 == Buffer : "
+            "1 == Texture)\", \"Byte Size\", \"Flags\"\n";
```

Comment:
Use raw C++ string will help avoid having to put all the escapes in front of the quotes:

R"("Pool Name", "Memory Type (0 == Host : 1 == Device)", "Allocation Name", "Allocation Type (0 == Buffer : 1 == Tex

Reply:
Actually, I'm realizing now that the quotes may not even be necessary since I think CSV permits spaces between words within a column